src/viewer.js

```diff
@@ -496,11 +559,6 @@ $.Viewer = function( options ) {
         beginControlsAutoHide( _this );
     } );
 
-    // Initial canvas options
-    if ( this.imageSmoothingEnabled !== undefined && !this.imageSmoothingEnabled){
```

Comment:
I don't see where this logic has been moved elsewhere, so it looks like it's just been removed? I figure we still want it.

Reply:
I think this was initially going to be a drawer-specific option but ended up not being implemented up there. I added it back in without the `if()` check - the drawer will deal with whatever value is passed to it.